tests/propertylookup/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -24,4 +31,14 @@ $b = $this->{'foo' . $bar};
 $b = $this->{self::STUFF};
 $object = $this->{'bar' . $foo}->buzz();
 
+$this->loooooooooooongloooooooooooongloooooooooooonglookup =
+    "teeeeeeeeeeeeeeeeeeeeest";
+$this->loooooooooooong->loooooooooooong->loooooooooooong->lookup =
+    "teeeeeeeeeeest";
+$this->loooooooooooong->loooooooooooong->loooooooooooong->lookup =
+    $other->looooooooooong->stuff;
+$this->loooooooooooong->lookup = (int) $this->getRequest()->getParam(
```

Comment:
Maybe also better `(int) $this`: https://prettier.io/playground/#N4Igxg9gdgLgprEAuEASGALAlgZwHQA2ExJpxUA5ocQNYCuADgAQC8TT62+FcMASnACOdODhgAKAJR4eMAAoBDAE4KAtuIA6UduwDkWKGIVQwcALRLREOktNmsAE11bJAbhAAaEBAYws0HGRQZSUIAHdFS0NkEAUANwhHT1ixZAAzBQIcOC8AIxUwGl4AZQYFMAMKZBglES8HCDB0zOyvA2yleRUKVQVmrJyQACscAA8AIQKimGK1OAAZAzh+1u86GAZ1gCYVwbKlDpjchVyATyIoZIYlAxgAdUdMZAAOAAYva4hsu5UGGOvRHAlHFll5LMIsJY5N1ersvNlVFhqrVBjhKgQ4ABFOgQeBwkAwE4PBxPJBbLw1BRYAiVADCEFUsJQUGgoJAdGyABUToEkBkBgBfAVAA

Reply:
You're right, this should have been picked up by the chained printing function - let's fix that in another PR, ok?

Reply:
@czosel just create issue about this, and let's do it in another PR.